packages/asc-ui/src/components/shared/sizes.ts

```diff
@@ -0,0 +1,9 @@
+export const HEADER_MAX_WIDTH = 1600
```

Comment:
these sizes are specific for atlas, perhaps a good time to refactor now? They shouldn't be in asc..

Reply:
These are indeed the implementation for atlas. We'll look into it.